openapi/api.yml

```diff
@@ -0,0 +1,61 @@
+openapi: "3.0.0"
```

Comment:
didnt know where to put this comment but as openapi/ and api/ directories are linked - should we reorganise our route structure to reflect this ?

Reply:
Yeah good thought. Though it raises a question of how much our repos should all strictly follow this same directory structure. The majority of repos are either an api(store) or a frontend, although some have both. I was thinking this template should cater for both, but ones that are just apis tend to have the openapi yamls in one root folder and then application code in /core or /app, so was kind of following that...?

Reply:
maybe we could have two directories, an /api directory and an /app directory?

Reply:
I think /api/openapi/api.yml makes sense. Possibly a good discussion to have team wide around directory structure across repos 👍

Reply:
Ok lets add to agenda for the RSU